.github/workflows/build-and-deploy.yml

```
@@ -44,6 +49,17 @@ jobs:
       run: |
         mdbook build
 
+    - name: Build IT translation
+      working-directory: gb-asm-tutorial/
+      env:
+        MDBOOK_BUILD__CREATE_MISSING: "false" # Prevent creating missing files in SUMMARY.md
+        MDBOOK_OUTPUT__LINKCHECK__COMMAND: "../mdbook-linkcheck"
+        MDBOOK_OUTPUT__LINKCHECK__OPTIONAL: "false"
+        MDBOOK_BOOK__LANGUAGE: "it"
+      run: |
+        mdbook build -d book/it
+        mv book/it/custom book/custom/it
```

```suggestion
        mdbook build
        for f in po/*.po; do
            lang="${f%.po}"
            MDBOOK_BOOK__LANGUAGE="$lang" mdbook build -d book/"$lang"
            mv book/"$lang"/custom book/custom/"$lang"
        done
```
This allows adding new languages without touching the CI workflow, and is also a bit more DRY.